### ISSUE TITLE
Able to look at more than 500 PRs?

### ISSUE THREAD

**Issue by jeep** [jeffpriz/devops-pr-stats#16]
We have a very active code base and we do 500 PRs in approximately 15 days. This appears to not look back to more than 500 PRs even if I pick 30 days.

**Comment by jeffpriz** [comment]
yeah, i've got it kind of limited to the last 500 PRs.. that's how it's working right now.  I could extend that to pull back multiple series of 500 PRs... I may default it to go back and grab the last 2500 or something? but give people an option to "limit for performance" to the last 500.. the 500 limit really was only there to help limit the bulk of data we were retrieving.
